preprocess_test.py

```diff
@@ -0,0 +1,90 @@
+# Lint as: python3
+"""Tests for evaluating preprocessing."""
+
+import unittest
+from typing import List
+import preprocess
+
+class TestPreprocess(unittest.TestCase):
+    """Tests for evaluating preprocess.py."""
+
+
+    def test_preprocess_ud(self):
+        'Test loading in a ud data file.'
+        infile = "testdata/test_wo_ud_input.txt"
+        try:
+            input_text: List[str] = preprocess.process_data(infile, "ud")
+        except Exception:
+            input_text = None
```

Comment:
I'm not sure if you need to test IO in every unit test, the last one should be fine.

Reply:
That is true, since we _know_ that the other inputs exist, so we can simplify the other tests.